Classes/BEMCheckBox.m

```diff
@@ -221,9 +221,13 @@ - (void)handleTapCheckBox:(UITapGestureRecognizer *)recognizer {
 #pragma  mark - Helper methods -
 
 #pragma mark Increase touch area
-- (BOOL) pointInside:(CGPoint)point withEvent:(UIEvent *)event;
+- (BOOL)pointInside:(CGPoint)point withEvent:(UIEvent *)event;
 {
-    BOOL found = [super pointInside:point withEvent:event];
+    BOOL found = NO;
+    
+    if ((found = [super pointInside:point withEvent:event])) {
```

Comment:
There is an extra set of parenthesis on that if statement.

Reply:
@Boris-Em, The extra set of parenthesis is necessary , because we use assignment not comparison in `if` condition ,leads to compiler warning .
Whatever, I modify the statements order .